### PR TITLE
Checkbox (Wearable):  update according to guideline

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/700
[Problem] Example at wrong path, update style for checkbox.
[Solution] Add example to correct path and update it
	according to guide.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
If checkbox is not in the middle of screen, its size is wrong 50x50 but according to guide (html) should be 53x53. This size comes from transform operation calculated by JS. Similar issue for ui-arc-listview-5 is already reported https://github.com/Samsung/TAU/issues/905.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				</li>
+
+				<!-- TODO: Add more examples -->
+
+			</ul>
+		</div>
+	</div>
+	<script src="../../../js/app.js">
+	</script>
+	<script src="../../../js/lowBatteryCheck.js">
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html:41]: could you make sure if this is JS is really needed?

**Review by lslachciak — CHANGES_REQUESTED** [review]
this example doesn't work properly (check on/off is not changed) if accessed using menu items both on simulator and in browser.
If I open directly in browser example it works

see http://localhost:8080/examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html
vs.
http://localhost:8080/examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html [hunk 2]
```diff
@@ -18,40 +17,46 @@
...
 					</label>
 				</li>
 			</ul>
 		</div>
 	</div>
+	<script src="../../../js/app.js">
+	</script>
+	<script src="../../../js/lowBatteryCheck.js">
+	</script>
+	<script>
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html]: the problem with not working script when accessed from main menu(about which I wrote in previous comment) may be that you are not including this inside of 
`<div class="ui-page" data-go-to-top-button="true">`

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
 <html>
 
 <head>
 	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
 	<title>
-		Wearable UI
+		Checkbox
 	</title>
 	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html:10]: Don't remove the  'media="all and (-tizen-geometric-shape: circle)"'

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less [hunk 4]
```diff
@@ -186,12 +188,12 @@ input[type="radio"] {
 				width: 100%;
 
 				div + input {
-					margin-left: 10 * @unit_base;
+					margin-left: 8 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:191]: ditto.

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less [hunk 5]
```diff
@@ -165,6 +165,8 @@ input[type="radio"] {
 /** CheckBox / Radio button on arc-listview **/
 .ui-arc-listview-carousel-item.ui-listview {
 	li {
+		padding-left: 30 * @px_base;
+		padding-right: 16 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:169]: In wearable profile, we use the @unit_base by default.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Button
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html]: ditto.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html (hunk 3 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html:10]: They cause wrong layout on Browser and we already started to get rid of them.

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less (hunk 5 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:169]: Recently we started using @px_base in Wearable profile as well in consistency with Mobile.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html (hunk 3 above)
Comment (HunseopJeong) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html:10]: TAU currently supports Square and Circle types. If you delete this, it only applies to the Circle, so I don't think it is correct.
@lslachciak, Is it right?

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less (hunk 5 above)
Comment (HunseopJeong) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:169]: Okay, I see.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html (hunk 1 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html:41]: Most examples are using. I think it's better to add it to new examples as well with consistency.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html (hunk 2 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html]: Yes, the problem is that while navigation only page is taken into account, others elements are not.

File: examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html (hunk 3 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html:10]: we need to discuss this issue

Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html:10]: Since it's not decided yet I restored media query.

**Review by lslachciak — APPROVED** [review]
works fine now

**Pull request merged**